tests/test_status.py

```diff
@@ -27,3 +29,176 @@ def test_disk_space_status(disk_usage, kwargs, total_blocks, free_blocks, expect
     disk_usage.return_value = statvfs_fixture(total_blocks, free_blocks)
 
     assert get_disk_space_status(**kwargs) == expected_status
+
+
+def additional_check(key, value):
+    m = mock.MagicMock()
+    m.return_value = {key: value}
+    return m
+
+
+@pytest.mark.parametrize('disk_status, '
+                         'data_api_status, '
+                         'search_api_status, '
+                         'ignore_dependencies, '
+                         'additional_checks, '
+                         'expected_json, '
+                         'expected_http_status',
```

Comment:
You are actually able to just specify this as a sequence rather than a weird concatenated string (I found out recently)